pubspec.yaml

```diff
@@ -10,7 +10,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  flutter_svg: ^0.19.2+1
+  flutter_svg: ^0.19.0
```

Comment:
Why reduce the SVG version?  It would be nice to depend on the nullsafety for 4.0.0

Reply:
I would recommend only bumping the major version, this way it stay compatible a longer time.

For nullsafety, you are right, maybe you should made a branch dedicated to 4.0.0 ?